source/administration/config-settings.rst

```diff
@@ -60,10 +60,11 @@ Default Client Language ``"DefaultClientLocale": "en"``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Default language for newly created users and pages where the user hasn't logged in.
 
-Available Languages ``"AvailableLocales": "en,es,fr,ja,pt-BR"``
+Available Languages ``"AvailableLocales": ""``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  
-Sets which languages are available for users in **Account Settings** > **Display** > **Languages**. Add new languages using the dropdown menu as they become available.
+Sets which languages are available for users in **Account Settings** > **Display** > **Languages**. Leave the field blank to add new languages automatically by default, or add new languages using the dropdown menu manually as they become available.
```

Comment:
We should add a note here that if  you're setting them manually, the default client language has to be selected here or else you can't save the setting